script.js

```diff
@@ -98,19 +98,41 @@ function makeTmdbAjaxCall(movie) {
                  */
                 var moviePoster = "http://image.tmdb.org/t/p/original" + movieData.backdrop_path;
                 $("#divForMovieInfo").empty();
-                $("<img>").attr({
-                    src: moviePoster
-                }).appendTo("#divForImage");
-                $("<h1>").text(movieData.original_title).appendTo("#divForMovieInfo");
-                $("<h2>").text(movieData.tagline).appendTo("#divForMovieInfo");
-                $("<h3>").text(movieData.release_date).appendTo("#divForMovieInfo");
-                $("<p>").text(movieData.overview).appendTo("#divForMovieInfo");
+                $("main").css('background-image', 'url(' + moviePoster + ')');
+                $("<h1>").text(movieData.original_title).appendTo("#divForMovieTitle");
+                $("<h2>").text(movieData.tagline).appendTo("#divForSummary");
+                $("<h3>").text("Released: " + movieData.release_date).appendTo("#divForSummary");
+                $("<p>").text(movieData.overview).appendTo("#divForSummary");
             }
         })
     })
 }
 
 /**
+ * url - empty string to be defined later
+ * @type {string}
+ */
+var url = "";
+/**
+ * Function to start the ajax call to itunes
+ */
+function searchItunes(search) {
+    url = "https://itunes.apple.com/search?media=music&order=popular&term=" + search + " soundtrack&callback=?";
+    $.getJSON(url, function (data) {
+        $('#musicSrc').attr('src', data.results[0].previewUrl);
+        $('#musicImg').attr('src', data.results[0].artworkUrl100);
+        $('#artistName').text(data.results[0].artistName);
+        $('#music').attr('src', data.results[0].previewUrl);
+    });
+}
+/**
+ * music - variable to create music player
+ * @type {Element}
+ */
+var music = document.getElementById("music");
```

Comment:
The following lines about the music player have been found to not be necessary. Can you remove them from your copy and double check that everything still works.